### ISSUE TITLE
Possible wrong conversion for msftyz 

### ISSUE THREAD

**Issue by aearamos** [EC-Earth/ece2cmor3#432]
Hi @goord and @treerink,
After comparing my raw output and my cmorized output for variable msftyz, I noticed something odd. 

For one of my files, my raw output has a range of -6.44853e+10 to 1.27822e+11, while my cmorized output has a range of -6.44853e+19 to 1.27822e+20. Both values are in kg s-1.

I noticed the nemopar.json and there's no conversion there for msftyz, such as the one present in nemo2cmor.py:

 if conversion == "sv2kgps":
        return 1.e+9, 0.

Since we're now outputing the variable as kg s-1, there is no need for an extra conversion, right? Because it seems to me that it's being multiplied by e+9 even if it's not in the nemopar file. 

Thanks!

**Comment by treerink** [comment]
Hi @aearamos, I first had a look in the code, could not find something. Thereafter I compared the value range of my raw output with the one of the cmorised data for this variable `msftyz`, but I find the same ranges. So I do not confirm/encounter your issue? 

I compared a first leg of a run in which I used the
`````
ctrl/cmip6-output-control-files/test-all-nemo-mip-variables
`````
which is a piControl run with all NEMO output. I cmorised with a very recent ece2cmor3 version.
